idaes/gas_solid_contactors/unit_models/moving_bed.py

```diff
@@ -38,7 +38,7 @@
 # Import Pyomo libraries
 from pyomo.environ import (Var, Param, Reals, value,
                            TransformationFactory, Constraint,
-                           check_optimal_termination)
+                           TerminationCondition, units as pyunits)
```

Comment:
@bpaul4 This change needs to be reverted-  `check_optimal_termination` is the correct way to check solver convergence as it is less solver specific.